apim/3.x/templates/api/api-deployment.yaml

```diff
@@ -107,6 +107,10 @@ spec:
           ports:
             - name: {{ .Values.api.service.internalPortName }}
               containerPort: {{ .Values.api.service.internalPort }}
+            {{- if .Values.api.http.services.core.http.enabled }}
+            - name: {{ .Values.api.name }}-hook
```

Comment:
Not sure what's the meaning of `-hook` here. Shouldn't it be something like `-tech-api` ?

Reply:
To be consistent with https://github.com/gravitee-io/helm-charts/blob/70d07a94244a55a79879cba445100b610cf977e4/apim/3.x/templates/gateway/gateway-deployment.yaml#L128

Reply:
Ok, I think that doesn't make sense. This one has been added in the past to the gateway for an other purpose (kubernetes hook), and I think it's been done wrongly at that time.